pkg/eventstore/eventstore.go

```diff
@@ -0,0 +1,22 @@
+package eventstore
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ThreeDotsLabs/esja/pkg/aggregate"
+)
+
+var (
+	ErrAggregateNotFound = errors.New("aggregate not found by ID")
+)
+
+// EventStore loads and saves T implementing aggregate.Aggregate.
+//
+// An example implementation of EventStore:
+// 1. Load would fetch all events for `AggregateID()` and use them to instantiate a `T` using `FromEvents()` and return it.
```

Comment:
- `Load would fetch all events for `AggregateID()`
+ `Load would fetch all events for `id`